Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -3,7 +3,7 @@
 # This object is used to selectively make changes for different running
 # scenarios. In this case it makes changes for Run 2.
 
-from EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi import *
+from EventFilter.SiPixelRawToDigi.siPixelDigis_cff import *
```

Comment:
No, it is part of the topic of this PR. The change is needed because `siPixelDigis` is no longer produced by a single producer, but by multiple producers. (Technically this change breaks running `cmsDriver.py -s RAW2DIGI` alone, that should probably be fixed even if it would mean importing from `RecoLocalTracker.SiPixelClusterizer` in `EventFilter.SiPixelRawToDigi`...)